values.yaml

```diff
@@ -63,6 +63,44 @@ web:
     #      - chart-example.local
 
 server:
+  image:
+    repository: philipssoftware/dctna-server
+    pullPolicy: IfNotPresent
```

Comment:
It is good practice to encapsulate this field as variable.

Reply:
How do you mean? This is the values file.

Reply:
I mean the IfNotPresent is hard coded, for other templates you use a variable and set the default in values.yml